hyp3_rtc_gamma/rtc_sentinel.py

```diff
@@ -657,6 +657,8 @@ def rtc_sentinel_gamma(in_file,
     report_kwargs(in_file, out_name, res, dem, roi, shape, match_flag, dead_flag, gamma_flag, lo_flag,
                   pwr_flag, filter_flag, looks, terms, par, no_cross_pol, smooth, area, orbit_file)
 
+    orbit_file = os.path.join(os.getcwd(), orbit_file)  # ingest_S1_granule requires absolute path
```

Comment:
might be more explicit to do an `os.path.abspath`

Reply:
and could probably me moved into your helper `fetch_orbit_file` function

Reply:
I prefer handing just the filename to `get_product_name` and `report_kwargs` (for now).  Implemented `os.path.abspath()`.